### DISCUSSION TITLE
what if analysis with the optimization

### DISCUSSION THREAD

**Discussion by hellojungkook** [google/lightweight_mmm#147] [Ideas]
Hi 
I just wonder whether there is a way to run a simulation with the optimized budget to get the expected revenue or sales?
I am trying to persuade client with this model but I need to deliver the budget suggestion with the expectation.
let say if you spend $xxx or 10% on YouTube  then you may see $xxx or %xxx lift in the sales.
Thank you in advance!

**Comment by steven-struglia** [comment]
This would be amazing

**Comment by adinamarca** [comment]
Yes please!
